src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -231,44 +211,34 @@ class ItemPage extends React.Component {
               aria-relevant="additions removals"
               aria-describedby="results-description"
             >
-              <div className="nypl-results-item">
+              <div className="nypl-item-details">
                 <h1>{title}</h1>
-
-                <ul className="nypl-item-toc">
-                  <li><a href="#item-holdings">Item holdings</a></li>
-                  <li><a href="#item-details">Item details</a></li>
-                  <li><a href="#item-external-links">External links</a></li>
-                </ul>
-
-                <div id="nypl-results-text">
-                  <ItemOverview
-                    data={itemOverview}
-                  />
-                </div>
-
-                <ItemHoldings
-                  path={this.props.location.search}
-                  holdings={holdings}
-                  title={`${record.numAvailable} item${record.numAvailable === 1 ? '' : 's'} associated with this record:`}
-                />
-
-                <div className="item-details">
-                  <div id="item-details">
-                    <ItemDetails
-                      data={itemDetails}
-                      title="Item details"
-                    />
-                  </div>
-
-                  <div id="item-external-links">
-                    <ItemDetails
-                      data={externalLinks}
-                      title="External links"
-                    />
-                  </div>
+                <div className="nypl-item-info">
+                  <p>
+                    <span className="nypl-item-media">{materialType}</span>
+                    {language && ' in ${language}'}
+                  </p>
+                  <p>{record.extent} {record.dimensions}</p>
+                  <p>
+                    {record.placeOfPublication}
+                    {record.publisher} {yearPublished}
+                  </p>
+                  <p>{language && ' in ${language} (how to get there?)'}</p>
```

Comment:
Hahah I just opened this page up to say -- actually I am gonna remove this entirely for now and make a new ticket since we don't have this mapped out.